diff --git a/node_modules/@capacitor/app/ios/Sources/AppPlugin/AppPlugin.swift b/node_modules/@capacitor/app/ios/Sources/AppPlugin/AppPlugin.swift
index c93b84a..6e73832 100644
--- a/node_modules/@capacitor/app/ios/Sources/AppPlugin/AppPlugin.swift
+++ b/node_modules/@capacitor/app/ios/Sources/AppPlugin/AppPlugin.swift
@@ -118,7 +118,7 @@ public class AppPlugin: CAPPlugin, CAPBridgedPlugin {
 
     @objc func getAppLanguage(_ call: CAPPluginCall) {
         call.resolve([
-            "value": Bundle.main.preferredLocalizations.first
+            "value": Bundle.main.preferredLocalizations.first ?? ""
         ])
     }
 
