diff --git a/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m b/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
index 54a6694..c097dd9 100644
--- a/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
+++ b/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
@@ -45,8 +45,10 @@ typedef enum : NSUInteger {
 
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wprotocol"
+#pragma clang diagnostic ignored "-Wobjc-protocol-property-synthesis"
 // suppressing warnings of the type: "Class 'KeyboardPlugin' does not conform to protocol 'CAPBridgedPlugin'"
-// protocol conformance for this class is implemented by a macro and clang isn't detecting that
+// and "Auto property synthesis will not synthesize property 'identifier'/'jsName'/'pluginMethods'"
+// protocol properties are implemented by the CAP_PLUGIN macro; clang isn't detecting that
 @implementation KeyboardPlugin
 
 NSTimer *hideTimer;
